src/main/java/buildcraft/additionalpipes/item/ItemDogDeaggravator.java

```diff
@@ -40,8 +40,8 @@ public ActionResult<ItemStack> onItemRightClick(World world, EntityPlayer player
     {
 		//this code adapted from EntityAIHurtByTarget.startExecuting()
 		double horizontalRange = 16;
-        List<EntityWolf> list = world.getEntitiesWithinAABB(EntityWolf.class, new AxisAlignedBB(player.posX, player.posY, player.posZ,
-        		player.posX + 1.0D, player.posY + 1.0D, player.posZ + 1.0D).grow(horizontalRange, 10.0D, horizontalRange));
+        List<EntityWolf> list = world.getEntitiesWithinAABB(EntityWolf.class, new AxisAlignedBB(player.posX, player.posY + 1.0D, player.posZ,
```

Comment:
Randomly noticed that this logic is slightly broken. If we want to create the bounding box _centered_ on the player, then it should be initially created with size of 0 and centered on the player's coordinates.